### ISSUE TITLE
ontology terms should render as links

### ISSUE THREAD

**Issue by tgbugs** [MetaCell/sds-viewer#134]
Ontology labels are currently being converted to chits that are label only. This is confusing because we want to know that those were controlled terms with some deeper meaning. One solution would be to make the chits that come from controlled terms into clickable chits that will show the url on hover and open the ontology term url in a new tab.

**Comment by tgbugs** [comment]
Taking a look at the results, it seems that the clickable chips are present, but both the label and url are empty in some cases?
![sds-viewer-chip-bug-1](https://user-images.githubusercontent.com/4299776/184982425-31013785-a2e2-477e-8e0e-37c68fafa0c6.jpg)
t

**Comment by jrmartin** [comment]
cc @ddelpiano

**Comment by tgbugs** [comment]
@ddelpiano with the latest version the chips in the About section are clickable and take the user to the url, but the same chip on the subject/sample level is not (though it is now showing the label).

**Comment by ddelpiano** [comment]
> @ddelpiano with the latest version the chips in the About section are clickable and take the user to the url, but the same chip on the subject/sample level is not (though it is now showing the label). See for example 2f3b0cf8-6af9-4976-b934-3010a6ef1137.

Hi @tgbugs , for this issue I had a chat internally with @jrmartin and I highlighted that imho this should be done in a more structural way doing some changes to our internal model so that this is done once for all the properties rather than cherry-picking which one should be clickable and which not.
Unfortunately at this last round I did not have enough time to fit this as described above and in order to address what requested in our call I did this for the specific fields you highlighted. I am sure I can put some more time to extend this for these fields if a high priority, but again this will be a cherry-picking implementation on specific fields rather than a generic implementation.

Thanks!
